services/admin/src/server.ts

```diff
@@ -176,26 +242,41 @@ export function buildApp({ store }: { store: Store }): Application {
     }
   );
 
-  app.get<NoParams>(
-    '/admin/write-ins/adjudications/contestId/count',
+  app.get<ElectionsNamespaceParams>(
+    '/admin/elections/:electionId/adjudications/grouped-by/contest-id',
     (_, response) => {
       response.json(store.getAdjudicationCountsGroupedByContestId());
     }
   );
 
-  app.get<NoParams>('/admin/write-ins/adjudications/counts', (_, response) => {
-    response.json(store.getAdjudicationCountsByContestIdAndTranscribedValue());
-  });
+  app.get<ElectionsNamespaceParams>(
+    '/admin/elections/:electionId/adjudications/grouped-by/contest-id-and-transcribed-value',
+    (request, response) => {
+      response.json(
+        store.getAdjudicationCountsByContestIdAndTranscribedValue(
+          request.params.electionId
+        )
+      );
+    }
+  );
 
-  app.get('/admin/write-ins/adjudications/:contestId/', (request, response) => {
-    const { contestId } = request.params;
-    response.json(store.getAdjudicationsByContestId(contestId));
-  });
+  app.get<ContestNamespaceParams>(
+    '/admin/elections/:electionId/contests/:contestId/adjudications',
```

Comment:
After designing an API for Arlo that used nested resources like this, I ended up finding it kind of overly verbose and annoying. I now lean more towards less nesting (e.g. `/admin/contests/:contestId/adjudications` or `/admin/elections/:electionId/adjudications?contest_id=<id>`). Not a big deal if you prefer this but thought I'd mention it (I know this is prime bike-shed territory 😄 )

Reply:
I buy this.

Reply:
These API endpoints are now removed, but I'll keep this in mind as I add new ones back in.